### PR TITLE
Add instruction document

### LINKED ISSUE #1: Need an instruction document for others to try this website
The document should have step-by-step instructions to start this website from scratch. For now, we don't need deployment instructions. 

- [x] have a fresh copy of this repository
- [x] start the website in this fresh copy and write every step down
- [x] format the document so that it's easy to follow

### PR DESCRIPTION
**Pull request by sjiang1**
I added a README.md to describe how to start the website for development.

The README.md includes:
1. a title "Movies Website for COSC 381"
2. a section for steps to start the website, which talks about:
    a. virtual environment
    b. pip install dependencies
    c. set up the environment variables for the flask app
    d. initialize the app's database and start the flask app

issue #1

### CODE REVIEW

**Comment by aaleryani99** [comment]
We probably should add a step about deactivating the virtual environment.

**Comment by Nick-Garska** [comment]
why is it labeled a a a b c d and 1 2

File: README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Movies Website for COSC 381
+## Steps for starting the website (development mode)
+
+1. Create a virtual environment in the root directory: `python3 -m venv venv`
+2. Activate the virtual environment: `source venv/bin/activate`
```
Comment (nstringham) [inline README.md:5]: you could add windows instructions

**Review by nstringham — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+3. Install dependencies: `python3 -m pip install -r requirements.txt`
+4. Set up environment variables for the flask app: `source env-var.sh`
+5. Initialize the database: `flask init-db` After the database is initialized, the database file is at: `instance/flaskr.sqlite`
+6. Start the website: `flask run`
+7. The webpage can be checked at: http://127.0.0.1:5000/movies/
```
Comment (sjiang1) [inline README.md:10]: We probably should add a description about deactivating the virtual environment. And, how to stop the flask app.

**Comment by sjiang1** [comment]
The commit a828c43 resolved the previous review/comment.

**Pull request merged**